openstackgit-update.yml

```diff
@@ -23,7 +23,7 @@
 # is expected to contain more repositories than RPC will need, but
 # will always have a complete set.
 
-- name: Update the openstackgit clones
+- name: Fetch the latest openstackgit clones
   hosts: localhost
   connection: local
```

Comment:
you don't seem to include artifact version, which should be in artifacting-vars.yml